Validate content against master 0.9.59+

```
10:00:23 [ERROR] BUILD FAILURE
10:00:23 [ERROR] ------------------------------------------------------------
10:00:23 [ERROR] CloudSlang build for repository: "/home/ubuntu/cloud-slang-content" failed due to failed tests.
10:00:23 [ERROR] Following 28 tests failed:
10:00:23 [ERROR] - Test case failed: testSortListStrSuccess - Tests that sort_list.sl operation finishes with result = ['element1', 'element2', 'element3', 'element4', 'element5', 'element6', 'element7']
		For output: result
		Expected value: ['element1', 'element2', 'element3', 'element4', 'element5', 'element6', 'element7']
		Actual value: [element1, element2, element3, element4, element5, element6, element7]
10:00:23 [ERROR] - Error occurred while running test: testRemoveObjectFromJsonArraySize2 - Tests that remove_object_from_json_array.sl operation finishes with SUCCESS for a valid command
		Error occurred during navigation execution  , Error running: 'equals'.
		Error evaluating result: 'EQUALS',
		Error is: Error in running script expression: 'return_code == '0' and decoded1 == decoded2',
		Exception is: Automatic proxy initialization should only occur on proxy classes
```

--- a/.travis.yml
+++ b/.travis.yml
@@ -29,7 +29,7 @@ script:
   - docker info
   - docker version
   - df -kh
-  - wget https://github.com/CloudSlang/cloud-slang/releases/download/cloudslang-0.9.58/cslang-builder.zip
+  - wget https://github.com/CloudSlang/cloud-slang/releases/download/cloudslang-0.9.59.1/cslang-builder.zip
   - unzip -q cslang-builder.zip -d cslang-builder
   - chmod +x cslang-builder/bin/cslang-builder
   - ./cslang-builder/bin/cslang-builder -ts ${SUITE},\!default -cov

--- a/circle.yml
+++ b/circle.yml
@@ -65,7 +65,7 @@ test:
         fi
       : parallel: true
     - ? > ### every machine
-        wget https://github.com/CloudSlang/cloud-slang/releases/download/cloudslang-0.9.58/cslang-builder.zip
+        wget https://github.com/CloudSlang/cloud-slang/releases/download/cloudslang-0.9.59.1/cslang-builder.zip
         && unzip cslang-builder.zip -d cslang-builder
         && chmod +x cslang-builder/bin/cslang-builder
         && mkdir cslang-builder/lib/Lib

--- a/content/io/cloudslang/cloud/openstack/servers/create_server_flow.sl
+++ b/content/io/cloudslang/cloud/openstack/servers/create_server_flow.sl
@@ -240,7 +240,7 @@ flow:
           - body_json: ${json_output}
           - return_result
           - return_code
-          - error_message:
+          - error_message
         navigate:
           - SUCCESS: add_availability_zone
           - FAILURE: ADD_BLOCK_DEVICE_MAPPING_FAILURE

--- a/content/io/cloudslang/consul/parse_register_agent_service_request.sl
+++ b/content/io/cloudslang/consul/parse_register_agent_service_request.sl
@@ -41,7 +41,7 @@ operation:
         data = {}
         if address:
           data['Address'] = address
-        if service_id != '':
+        if service_id:
           data['ID'] = service_id
         data['Name'] = service_name
         if check:

